src/content/en/updates/2020/05/heavy-ad-interventions.md

```diff
@@ -234,6 +230,11 @@ Frames**". If testing content in the top-level window or other context where it
 is not tagged as an ad the intervention will not be triggered, but you can still
 manually check against the thresholds.
 
+A frame's ad status is also displayed in the **Elements** pane where an `ad`
+annotation is added after the opening `<iframe>` tag. This is also visible in
+the **Application** panel under the **Frames** section, where ad-tagged frames
+will include an "**Ad Status**" attribute.
```

Comment:
Nit: does **Ad Status** need the quotes?

Reply:
This is for consistency with the rest of the article where, as far as I can see, they style has been that an end node, like a button or single attribute, do get quotes. e.g.

- **More Tools** > **Rendering**) select "**Highlight Ad Frames**"
- ensure the "**Disable cache**" option is checked